src/AI/ModularAI.java

```diff
@@ -27,6 +28,7 @@ public abstract class ModularAI implements AIController
   protected ArrayList<AIModule> aiPhases;
   // Sets the ordering for units in the unit queue fed to the modules
   protected Comparator<Unit> unitOrderSetter = new AIUtils.UnitCostComparator(false);
+  protected CapPhaseAnalyzer capPhase;
```

Comment:
Seems a little odd to have this as part of the base class here, instead of self-contained in the module. I guess that would require the Commander to initialize it once the map is available, but there's already an init step anyway.

Reply:
Fair point.
The most immediate reason is "it seemed simpler this way, and I don't foresee making new AIs that don't want to have a good cap phase".
The other reason is that the `CapPhaseAnalyzer` could have relevant data for things other than just the cap phase - e.g. what cities will be contested could feed into other computation.
I suspect that the second reason there is something I *was* going to use for JakeMan, then didn't actually do anything with it because Lost&Found specified that it be calculated and... then didn't use it later.

Reply:
Hah. Did you follow up to see why Lost&Found wanted to calculate that thing?

Reply:
Quoth he:
> The idea was to use neutral/opposing contested props as "where to go" tiebreakers pretty much - cities and the tiles next to them being extra spots to cover if it's possible to do that while not compromising the rest of your stuff
> But I suppose computers do what you tell them to and I am bad at talk

> Either what I just said or this was just step 1 of making your AI do my tax returns for me

I thought about trying to do that, but then I decided I already had something acceptable (copied from Wally) and he hadn't written up any sort of real algorithm on it.